modules/apiserversource-kn.adoc

```diff
@@ -83,16 +83,18 @@ Conditions:
 
 .Verification
 
-You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+To verify that the Kubernetes events were sent to Knative, look at the event-display logs or use web browser to see the events.
 
-. Get the pods:
+. To view the events in a web browser, open the link returned by the following command:
 +
 [source,terminal]
 ----
-$ oc get pods
+$ kn service describe event-display -o url
 ----
 
-. View the message dumper function logs for the pods:
+image::serverless-apiserversource-eventdisplay-gui.png[Example visualization of ApiServerSource event]
+
```

Comment:
Need a `+` to keep the image in line with the text.
```suggestion
+
```
The missing `+` also affects the numbering. See [Verification in the preview](https://60772--docspreview.netlify.app/openshift-serverless/latest/eventing/event-sources/serverless-apiserversource#apiserversource-kn_serverless-apiserversource)